src/bp/ui-admin/src/telemetry/client.ts

```diff
@@ -0,0 +1,146 @@
+import axios from 'axios'
+import { lang } from 'botpress/shared'
+import _ from 'lodash'
+import moment from 'moment'
+import ms from 'ms'
+import uuid from 'uuid'
+
+import store from '../store'
+
+import { EventPackageInfoType, StoreInfoType, TelemetryPackage } from './telemetry_type'
+
+const telemetryPackageVersion = '1.0.0'
+const dataClusterVersion = '1.0.0'
+
+export const axiosConfig = {
+  baseURL: window.TELEMETRY_URL,
+  headers: {
+    withCredentials: false
+  }
+}
+
+const storeInfos: StoreInfoType = {}
```

Comment:
why do you create an object with some loading and accessor and setters to play around with data that is already in a store that has all those capabilities ?  It feels like you added come complexity to your code juste for the sake of adding code ...

Reply:
Fixed in : e135fcf